packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -667,14 +668,14 @@ curl -v -X POST \
+| id             | Unique key for the Inline Hook                                                                     | String                              | FALSE      | TRUE     | TRUE       | System assigned                                          |
 | status         | Status of the Inline Hook. `INACTIVE` will block execution.                                         | String                              | FALSE      | FALSE    | FALSE      | System assigned. Will be either `ACTIVE` or `INACTIVE`.            |
-| name           | Display name for the Inline Hook.                                                                   | String                              | FALSE      | TRUE     | FALSE      | Must be between 1 and 255 characters in length.   |
-| type           | Type of the Inline Hook. See list of [Supported Inline Hook Types](#supported-inline-hook-types).   | inlineHookType                      | FALSE      | FALSE    | TRUE       | Immutable after Inline Hook creation.             |
-| version        | Version of the Inline Hook type. The currently-supported version is "1.0.0".                                 | String                              | FALSE      | FALSE    | TRUE       | Must match a valid version number.                |
-| channel | Properties of the communications channel used to contact your external service.                     | [Channel object](#channel-object)   | FALSE      | FALSE    | FALSE      | Validation is determined by the specific channel. |
-| created        | Date of Inline Hook creation.                                                                       | String (Date)                       | TRUE       | FALSE    | TRUE       | System assigned                                          |
-| lastUpdated    | Date of Inline Hook update.                                                                         | String (Date)                       | TRUE       | FALSE    | TRUE       | System assigned                                          |
+| name           | Display name for the Inline Hook                                                                   | String                              | FALSE      | TRUE     | FALSE      | Must be between 1 and 255 characters in length   |
+| type           | Type of the Inline Hook. See list of [Supported Inline Hook Types](#supported-inline-hook-types).   | inlineHookType                      | FALSE      | FALSE    | TRUE       | Immutable after Inline Hook creation             |
+| version        | Version of the Inline Hook type. The currently-supported version is "1.0.0".                                 | String                              | FALSE      | FALSE    | TRUE       | Must match a valid version number                |
```

Comment:
currently supported

The Chicago Manual of Style: “Compounds formed by an adverb ending in ly plus an adjective or participle (such as largely irrelevant or smartly dressed) are not hyphenated either before or after a noun, since ambiguity is virtually impossible.